src/zenml/repository.py

```diff
@@ -817,6 +824,14 @@ def deregister_stack(self, name: str) -> None:
             raise ValueError(f"Unable to deregister active stack '{name}'.")
         self.stack_store.deregister_stack(name)
 
+    def update_stack_component(
```

Comment:
Is this functionality (which I definitely think we should have) going to be part of this PR? Or can we delete it for the moment?

Reply:
I had forgotten to handle the use case of updating individual stack components. Adding now.